Object has no attribute 'storage', in size property on Django 3.1

I'm trying to upgrade from Django 3.0.10 to 3.1.2 and my test suite is throwing up this error:

```
self = <ThumbnailerImageFieldFile: REDACTED>

    @property
    def size(self):
        self._require_file()
        if not self._committed:
            return self.file.size
>       return self.storage.size(self.name)
E       AttributeError: 'ThumbnailerImageFieldFile' object has no attribute 'storage'
```

Is this my fault somehow? or is this a compatibility problem between easy-thumbnails and Django 3.1?

--- a/easy_thumbnails/files.py
+++ b/easy_thumbnails/files.py
@@ -735,7 +735,7 @@ class ThumbnailerFieldFile(FieldFile, Thumbnailer):
         return state
 
     def __setstate__(self, state):
-        self.__dict__.update(state)
+        super().__setstate__(state)
         self.__dict__['alias_target'] = self
 
 

--- a/easy_thumbnails/files.py
+++ b/easy_thumbnails/files.py
@@ -735,7 +735,10 @@ class ThumbnailerFieldFile(FieldFile, Thumbnailer):
         return state
 
     def __setstate__(self, state):
-        super().__setstate__(state)
+        if hasattr(super(), '__setstate__'):
+            super().__setstate__(state)
+        else:
+            self.__dict__.update(state)
         self.__dict__['alias_target'] = self
 
 

